lib/config.js

```diff
@@ -38,6 +38,11 @@ var Config = module.exports = function (cfg) {
   this.scripts = this._loadScripts(this.archetypes);
 };
 
+// Expose `require()` for testing.
```

Comment:
I understand why this is done after reading your tests, but can you elaborate in this comment a little more? On first read it's very unclear why this additional logic is here. Just mentioning something like "require needs to be stubbed to work with mock filesystem in tests" would be helpful.